string-manipulation-1/src/get-last-char.js

```diff
@@ -0,0 +1,10 @@
+/* exported getLastChar */
+function getLastChar(string) {
+  var lastString = '';
+  var lastIndex = '';
+  for (var i = 0; i < string.length; i++) {
+    lastString = string.length - 1;
+    lastIndex = string[lastString];
+  }
```

Comment:
You can grab the `string[string.length - 1]` without the loop.

Reply:
<img width="1242" alt="Screen Shot 2021-01-29 at 4 56 09 PM" src="https://user-images.githubusercontent.com/74885918/106341649-e4c6b480-6252-11eb-84b3-4690f39c6068.png">